NHLBI_BioData_Catalyst/python/1_PICSURE_API_101.ipynb

```diff
@@ -106,8 +110,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import sys\n",
-    "!{sys.executable} -m pip install -r requirements.txt"
+    "### Uncomment this code if you are not using the PIC-SURE environment in Seven Bridges, or if you do not have all the necessary dependencies installed.\n",
```

Comment:
Perhaps capitalizing "NOT" would be helpful. First time I read it I almost missed it. 
Could also put this as a separate markdown cell and bold "not" or something similar.

Reply:
Resolved in [c7ba961058342af5afe011bec51f38ef75a10139](https://github.com/hms-dbmi/Access-to-Data-using-PIC-SURE-API/commit/c7ba961058342af5afe011bec51f38ef75a10139)